proto/iot.proto

```diff
@@ -41,6 +41,13 @@ enum HVACState {
     HVACStateCoolStage2 = 4;
 }
 
+enum SystemMode {
+    SystemAuto = 0;
+    SystemHeat = 1;
+    SystemCool = 2;
+    SystemOff = 3;
+}
```

Comment:
Lets use the existing HVAC mode enum instead (https://github.com/gtfierro/xboswave/blob/master/proto/iot.proto#L29-L34)

Reply:
No problem! Deleted the state struct and replaced mode's type with HVACMode instead.